packages/inline-choice/configure/src/main.jsx

```diff
@@ -38,30 +41,42 @@ export default class Main extends React.Component {
     this.props.onChoiceChange(index, update);
   }
 
-  render() {
-
-    const usEnglishChoices = this.props.model.choices.map(flattener.flatten);
+  onPromptChange = (value) => {
+    this.props.onPromptChange([{ lang: 'en-US', value }]);
+  }
 
+  render() {
+    const { model, classes, onRemoveChoice, onAddChoice } = this.props;
+    const usEnglishChoices = model.choices.map(flattener.flatten);
+    const prompt = model.prompt.find(p => p.lang === 'en-US').value;
     return (
       <div>
-        {this.props.model.prompt && (
-          <MultiLangInput
```

Comment:
`MultiLangInput` is not needed as this config pane isn't going to support multiple langs for now.